spec/features/tags_spec.rb

```diff
@@ -187,5 +189,79 @@ def n_tagables(n, tagable_category)
         end
       end
     end
-  end
+
+    describe 'edits tab' do
+      let(:setup) { -> {} }
+      before(:each) do
+        setup.call
+        visit "/tags/#{tag.id}/edits"
+      end
+
+      it "has header with active edit tab" do
+        expect(page).to have_selector 'li#tag-nav-tab-edits.active'
+        expect(page).to have_selector 'li#tag-nav-tab-edits.active a', text: 'Edits'
+      end
+
+      it "contains list of edits" do
+        expect(page).to have_selector '#tag-homepage-edits-table-container table', count: 1
+      end
+
+      describe 'list of edits' do
+        let(:person) { create(:entity_person).tag(tag.id) }
+        let(:list) { create(:list).tag(tag.id) }
+        let(:relationship) do
+          create(:generic_relationship, entity: create(:entity_org), related: create(:entity_org)).tag(tag.id)
+        end
+        context 'a person was recently tagged' do
+          let(:setup) { proc { person } }
+          edits_table_has_correct_row_count(1)
+
+          it 'contains "tagged" text' do
+            expect(page.find('#tag-homepage-edits-table tbody')).to have_text 'tagged'
+            expect(page.find('#tag-homepage-edits-table tbody')).not_to have_text 'updated'
+          end
+        end
+
+        context 'a person was recently updated (and previously tagged)' do
+          let(:setup) { proc { person.update_column(:updated_at, Date.tomorrow) } }
+          edits_table_has_correct_row_count(2)
+
+          it 'contains "tagged" and "updated" text' do
+            expect(page.find('#tag-homepage-edits-table tbody')).to have_text 'tagged'
+            expect(page.find('#tag-homepage-edits-table tbody')).to have_text 'Entity updated'
+          end
+        end
+
+        context 'a list and a relationship were recently tagged' do
+          let(:setup) { proc { list; relationship; } }
+
+          edits_table_has_correct_row_count(2)
+
+          it 'contains links to the list and relationship' do
+            find('#tag-homepage-edits-table tbody') do |el|
+              expect(el).to have_link list.name
+              expect(el).to have_link relationship.name
+            end
+          end
+        end
+
+        xcontext 'a person was tagged, untagged, and then updated' do
```

Comment:
i did! but i probably need to explain myself. that tests passes as currently written and explains the current behavior-- so perhaps it should be un-skipped. 

We aren't versioning the Tagging model. Meaning, there's a record of when a person gets tagged (via the created_at field on taggings), but when a person is un-tagged, the tagging record is removed. So if a person is tagged and then un-tagged, no record will show up in the edits table that the person was __ever__ tagged. It would be possible to change this via paper_trail by keep track of the history of all taggings that ever get created and deleted. This would, however, add significant complexity to the query that generates this edits table and to the tagging model itself. So before we do that I think we should find out the benefit of keeping track of that information would actually be useful.